docs/contribute-module.md

```diff
@@ -13,12 +13,13 @@ Follow these steps to create or update a module.
 ## Before you begin
 
 - If you're running Microsoft Windows, set up Windows Subsystem for Linux [WSL](https://ubuntu.com/wsl) and run commands within WSL.
-- Make sure that you have a current version of Python 3 and pip.
+- Make sure that you have a current version of Python 3 and pip. 
+  - _Notes for the macOS users:_ The out-of-the-box Python (Python 2.x) which comes with macOS is not suitable. The installation instructions for the latest Python 3.x version on macOS can be found [here](python3-mac-installation.md).
 - Make sure that you have a recent stable version of [Go](https://go.dev/doc/install) installed and available in your PATH environment variable.
 
 ## Request a repo
 
-?> **Tip:** If you're updating an existing module, skip to the next step.
+**Tip:** If you're updating an existing module, skip to the next step.
```

Comment:
this needs to return to the previous syntax: `?>`. It creates the callout/admonition style.